[AMORO-2496][DOC]: Add configuration docs for AMS terminal (#2604)
NetEase/amoro: docs/admin-guides/deployment.md

| jmx.remote.port | "-Dcom.sun.management.jmxremote.port=${value}  | Enable remote debug      |
| extra.options   | "JAVA_OPTS="${JAVA_OPTS} ${JVM_EXTRA_CONFIG}"  | The addition jvm options |

### Terminal configurations

Terminal support local and kyuubi, the default is local. If the user uses local to run in the spark local context, you can set the **spark.*** configuration, and if you use kyuubi, you can set the **kyuubi.*** configuration

| Key                      | Default | Description                                                        |
|--------------------------|---------|--------------------------------------------------------------------|

> | terminal.backend         | local   | terminal backend implementation. local, kyuubi are supported       |

Reviewer: ```suggestion
| terminal.backend         | local   | Terminal backend implementation. local, kyuubi and custom are valid values.     |
```

> | terminal.factory         | -       | session factory implement of terminal                              |

Reviewer: ```suggestion
| terminal.factory         | -       | Session factory implement of terminal, `terminal.backend` must be `custom` if this is set.                              |
```

> | terminal.result.limit    | 1000    | limit of result-set                                                |

Reviewer: ```suggestion
| terminal.result.limit    | 1000    | Row limit of result-set                                                |
```

> | terminal.stop-on-error   | false   | if stop all statements in one execution when any statement failed. |

Reviewer: ```suggestion
| terminal.stop-on-error   | false   | When a statement fails to execute, stop execution or continue executing the remaining statements.  |
```

> | terminal.session.timeout | 30      | session timeout in minutes.                                        |

Reviewer: ```suggestion
| terminal.session.timeout | 30      | Session timeout in minutes.                                        |
```
